### DISCUSSION TITLE
Show terminal with a shortcut ?

### DISCUSSION THREAD

**Discussion by wiiznokes** [zed-industries/zed#17707] [Feature Requests]
So right now, i'm using this keymap to toggle the terminal:
```json
[
  {
    "bindings": {
      "ctrl-q": "workspace::ToggleRightDock"
    }
  }
]
```
This work really great, even better than vscode because it also work when the terminal is focused. __However it has one issue, each time i open zed, it show the assistant panel.__

This could be fixed by
- remembering the last opened panel
- provide a settings to set the default panel to open

**Comment by rohitshriwas** [comment]
It's ctrl-` to open the terminal on Linux. Open the default keymap and search for terminal_panel::ToggleFocus and that should be the shortcut for you. You can redefine it to a different key combination too.

**Reply by wiiznokes** [reply]
Thanks, though, that partially fix my problem. I would prefer to keep one shortcut for this

**Reply by rohitshriwas** [reply]
I wrote this in my keymap file to get a Ctrl-` to toggle the terminal pane:
```JSONC
  {
    "context": "Terminal",
    "bindings": {
      "ctrl-shift-t": "workspace::NewTerminal",
      "ctrl-`": "workspace::ToggleBottomDock"
    }
  }
```
Kind of like how it is in vscode. I have my terminal in the bottom dock though. You may need to make changes if the terminal is elsewhere for you.
